arg.c

```diff
@@ -29,10 +29,8 @@
  * 4. This notice may not be removed or altered.
  */
 
-#ifndef lint
-static char copyright[] =
+volatile static char copyright[] =
```

Comment:
So at least on my local build, gcc strips out this variable from the object file:

dilinger@e7470:/home/dev/lsof$ strings arg.o |grep Copy
dilinger@e7470:/home/dev/lsof$ 

With the standard caveat that I Am Not A Lawyer, the license text says "This notice may not be removed or altered", referring to the comment. The variable is not part of that comment or notice, and the text in the variable is duplicated in the comment. In my opinion, the variable is safe to remove. But I'm fine either way.

Reply:
Oh, I mean that gcc strips it out when the variable is non-volatile. I didn't test it with the volatile keyword added, but it looks okay to me.

Reply:
I introduced a change for printing the copyright notice to --version output.
Then I introduced another change deleting these variables.